contracts/minting/MasterMinter.sol

```diff
@@ -24,6 +24,10 @@ pragma solidity ^0.4.24;
 
 import "./MintController.sol";
 
+/**
+ * @title MasterMinter
+ * @dev Default implementation of the MintController contract.
```

Comment:
@notice MasterMinter uses multiple controllers to manage minters for a contract that implements the MinterManagerInterface.
@dev MasterMinter inherits all its functionality from MintController.